### DISCUSSION TITLE
Auto-insert hyperlinks

### DISCUSSION THREAD

**Discussion by avidseeker** [jgm/pandoc#9848] [Q&A] [answered]
Is there a lua script that automatically insert hyperlinks for text matching URLs or emails.

md2tex:
```
https://example.org -->  \href{https://me@example.org}{https://example.org}
me@example.org --> \href{mailto:me@somewhere.com}{me@somewhere.com}
```

md2html:
```
https://example.org -->  <a href="https://example.org">https://example.org</a>
me@example.org --> <a href="mailto:me@example.org">me@example.org</a>
```

**Comment by TomBener** [comment, marked as answer]
You need to enable [autolink_bare_uris](https://pandoc.org/MANUAL.html#extension-autolink_bare_uris) extension or use [automatic links](https://pandoc.org/MANUAL.html#automatic-links) syntax.
